v2/concordium/types.proto

```diff
@@ -2159,3 +2159,18 @@ message ElectionInfo {
   // List of the currently eligible bakers.
   repeated Baker baker_election_info = 3;
 }
+
+// A response consisting of a boolean.
+message BoolResponse {
+  bool value = 1;
+}
+
+// Request to enable dumping of network packages.
+message DumpRequest {
+  // Which file to dump the packages into.
+  // If this is not set then a default value of 'dump' will be used.
+  string file = 1;
```

Comment:
Can we just not have this default? I don't think it adds value, and makes the API a bit  more complicated. Just require a valid path.